panther_analysis_tool/detection.py

```diff
@@ -147,6 +147,28 @@ def detection_evaluation_failed(self) -> bool:
         """Returns whether the detection function raises an error or an import error occurred"""
         return bool(self.detection_exception or self.setup_exception)
 
+    def ignore_errors(self, ignore_exception_types: Optional[List[Type[Exception]]] = None) -> None:
+        """Used to ignore exceptions of particular types, used primarily in testing"""
+        for exception_type in ignore_exception_types or []:
```

Comment:
Minor: `isinstance` can accept an iterable as a second argument, the loop is not required.

Reply:
I think it has to be a tuple rather than a list (?) - so I can convert the list to a tuple in the check or make the change to make the list a tuple everywhere